python/ray/tests/test_logging.py

```diff
@@ -367,7 +373,7 @@ def matcher(log_batch):
 
     # NOTE(xychu): loop ten times to make job id from 01000000 to 0a000000,
     #              in order to trigger hex pattern
-    for _ in range(10):
+    for _ in range(5):
```

Comment:
why changing this. Also line 274 is not updated accordingly.

Reply:
Reverted, the latest CI passes.